README.md

```diff
@@ -20,7 +20,8 @@
 
```

Comment:
- [ ] You are using SQLite for your database, but you are required to use postgres for your database per the requirement. Kindly configure your database file to use postgres. You also need to install the `pg` gem.

  ![image](https://github.com/evansnyamekye/Blog-application/assets/54848363/ce4c38fc-7373-4ae5-8385-5de802459cff)